### PR TITLE
rebrand

### PR DESCRIPTION
**Pull request by CiotkaCierpienia**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ module.exports = {
         'react-hooks',
     ],
     rules: {
-        'linebreak-style': ['error', 'windows'],
+        // 'linebreak-style': ['error', 'windows'],
```
Comment (veto221) [inline .eslintrc.js:28]: Unwanted comment, better for me is remove this line

File: components/CustomBtn/styles.js [hunk 2]
```diff
@@ -3,12 +3,12 @@ import Colors from '../../helpers/constants/colors';
 
 const styles = StyleSheet.create({
     btn: {
-        backgroundColor: '#299BAC',
+        backgroundColor: Colors.primary,
```
Comment (veto221) [inline components/CustomBtn/styles.js:6]: Nice :) Better for me is using predefined colours instant of hash

File: components/CustomDrawer/styles.js [hunk 3]
```diff
@@ -10,8 +10,8 @@ const styles = StyleSheet.create({
     },
     uglyDrawerItem: {
         fontSize: 18,
-        fontFamily: 'Roboto-Bold',
-        color: '#000000',
+        fontFamily: 'Inter-Bold',
+        color: Colors.accent3,
```
Comment (veto221) [inline components/CustomDrawer/styles.js:14]: Nice :) Better for me is using predefined colours instant of hash

File: components/CustomListItem/CustomListItem.js [hunk 4]
```diff
@@ -41,7 +41,7 @@ const CustomListItem = React.memo(({
 
                             <Icon
                                 name="chevron-right"
-                                color="#000000"
+                                color="#050038"
```
Comment (veto221) [inline components/CustomListItem/CustomListItem.js:44]: Maybe there we can use predefined colour?

File: components/CustomListItem/SwipeableRow/styles.js [hunk 5]
```diff
@@ -1,9 +1,10 @@
 import { StyleSheet } from 'react-native';
+import Colors from '../../../helpers/constants/colors';
 
 const styles = StyleSheet.create({
     leftAction: {
         marginTop: 20,
-        backgroundColor: '#348fc0',
+        backgroundColor: Colors.primary,
```
Comment (veto221) [inline components/CustomListItem/SwipeableRow/styles.js:7]: Nice :) Better for me is using predefined colours instant of hash

File: components/CustomListItem/SwipeableRow/styles.js [hunk 6]
```diff
@@ -16,7 +17,7 @@ const styles = StyleSheet.create({
         marginTop: 20,
         alignItems: 'center',
         flexDirection: 'row',
-        backgroundColor: '#dd2c00',
+        backgroundColor: Colors.danger,
```
Comment (veto221) [inline components/CustomListItem/SwipeableRow/styles.js:20]: Nice :) Better for me is using predefined colours instant of hash

File: components/CustomListItem/styles.js [hunk 7]
```diff
@@ -19,14 +19,14 @@ const styles = StyleSheet.create({
     },
     title: {
         width: '85%',
-        fontFamily: 'Roboto-Medium',
+        fontFamily: 'Inter-Medium',
         fontSize: 18,
-        color: '#000000',
+        color: Colors.accent3,
```
Comment (veto221) [inline components/CustomListItem/styles.js:24]: Nice :) Better for me is using predefined colours instant of hash

File: components/CustomListItem/styles.js [hunk 8]
```diff
@@ -19,14 +19,14 @@ const styles = StyleSheet.create({
...
     },
     subtitle: {
-        fontFamily: 'Roboto-Regular',
+        fontFamily: 'Inter-Regular',
         fontSize: 16,
-        color: '#333333',
+        color: Colors.accent4,
```
Comment (veto221) [inline components/CustomListItem/styles.js:29]: Nice :) Better for me is using predefined colours instant of hash

File: components/FormModal/FormInput/styles.js [hunk 9]
```diff
@@ -1,23 +1,24 @@
...
+import Colors from '../../../helpers/constants/colors';
 
 const styles = StyleSheet.create({
     readOnly: {
         marginLeft: 15,
         marginTop: -20,
         fontSize: 12,
         marginBottom: 10,
-        color: '#82827E',
+        color: Colors.accent5,
```
Comment (veto221) [inline components/FormModal/FormInput/styles.js:10]: Nice :) Better for me is using predefined colours instant of hash

File: components/FormModal/FormInput/styles.js [hunk 10]
```diff
@@ -1,23 +1,24 @@
...
     },
     pickerContainer: {
         marginHorizontal: 10,
         marginTop: 0,
         marginBottom: 20,
         borderBottomWidth: 1,
-        borderBottomColor: '#86939E',
+        borderBottomColor: Colors.accent6,
```
Comment (veto221) [inline components/FormModal/FormInput/styles.js:17]: Nice :) Better for me is using predefined colours instant of hash

File: components/FormModal/FormInput/styles.js [hunk 11]
```diff
@@ -1,23 +1,24 @@
...
         padding: 0,
     },
     pickerLabel: {
-        color: '#86939E',
+        color: Colors.accent6,
```
Comment (veto221) [inline components/FormModal/FormInput/styles.js:21]: Nice :) Better for me is using predefined colours instant of hash

File: components/FormModal/RelationPicker/styles.js [hunk 12]
```diff
@@ -21,8 +22,8 @@ const styles = StyleSheet.create({
         marginBottom: 10,
     },
     pickerHeaderTxt: {
-        fontFamily: Platform.OS === 'ios' ? 'Roboto-Medium' : 'Roboto-Bold',
-        color: '#86939e',
+        fontFamily: Platform.OS === 'ios' ? 'Inter-Medium' : 'Inter-Bold',
+        color: Colors.accent6,
```
Comment (veto221) [inline components/FormModal/RelationPicker/styles.js:26]: Nice :) Better for me is using predefined colours instant of hash

File: components/FormModal/styles.js [hunk 13]
```diff
@@ -37,7 +37,7 @@ const styles = StyleSheet.create({
         paddingHorizontal: 10,
     },
     bottomSageArea: {
-        backgroundColor: '#000',
+        backgroundColor: Colors.accent3,
```
Comment (veto221) [inline components/FormModal/styles.js:40]: Nice :) Better for me is using predefined colours instant of hash

File: components/MenuDropdown/styles.js [hunk 14]
```diff
@@ -25,7 +26,7 @@ const styles = StyleSheet.create({
         width: 170,
         backgroundColor: '#fff',
         borderRadius: 2,
-        shadowColor: '#000000',
+        shadowColor: Colors.accent3,
```
Comment (veto221) [inline components/MenuDropdown/styles.js:29]: Nice :) Better for me is using predefined colours instant of hash

File: components/MenuDropdown/styles.js [hunk 15]
```diff
@@ -45,7 +46,7 @@ const styles = StyleSheet.create({
     },
     actionBtn: {
         backgroundColor: 'transparent',
-        borderColor: '#82827E',
+        borderColor: Colors.accent6,
```
Comment (veto221) [inline components/MenuDropdown/styles.js:49]: Nice :) Better for me is using predefined colours instant of hash

File: components/NoData/styles.js [hunk 16]
```diff
@@ -15,19 +16,19 @@ const styles = StyleSheet.create({
         marginTop: smallDeviceScreen ? 160 : 240,
     },
     title: {
-        fontFamily: 'Roboto-Light',
+        fontFamily: 'Inter-Light',
         fontSize: 24,
         letterSpacing: 1.5,
-        color: '#82827E',
+        color: Colors.accent6,
```
Comment (veto221) [inline components/NoData/styles.js:22]: Nice :) Better for me is using predefined colours instant of hash

File: components/Scanner/styles.js [hunk 17]
```diff
@@ -8,18 +8,18 @@ const styles = StyleSheet.create({
     },
     containerStyle: {
         width: '100%',
-        backgroundColor: '#000',
+        backgroundColor: Colors.accent3,
```
Comment (veto221) [inline components/Scanner/styles.js:11]: Nice :) Better for me is using predefined colours instant of hash

File: components/Scanner/styles.js [hunk 18]
```diff
@@ -8,18 +8,18 @@ const styles = StyleSheet.create({
...
     },
     botomContent: {
         color: Colors.secondary,
         fontSize: 20,
-        fontFamily: 'Roboto-Regular',
+        fontFamily: 'Inter-Regular',
     },
     scannerContainer: {
-        backgroundColor: '#000',
+        backgroundColor: Colors.accent3,
```
Comment (veto221) [inline components/Scanner/styles.js:22]: Nice :) Better for me is using predefined colours instant of hash

File: components/Search/styles.js [hunk 19]
```diff
@@ -15,7 +15,7 @@ const styles = StyleSheet.create({
         backgroundColor: 'transparent',
         borderTopWidth: 0,
         borderBottomWidth: 1,
-        borderBottomColor: '#000',
+        borderBottomColor: Colors.accent3,
```
Comment (veto221) [inline components/Search/styles.js:18]: Nice :) Better for me is using predefined colours instant of hash

File: components/Search/styles.js [hunk 20]
```diff
@@ -38,13 +38,13 @@ const styles = StyleSheet.create({
...
     searchBarInput: {
-        fontFamily: 'Roboto-Regular',
+        fontFamily: 'Inter-Regular',
         paddingBottom: 0,
         marginTop: 25,
         marginLeft: 0,
         paddingLeft: 0,
         textAlign: 'left',
-        color: '#000',
+        color: Colors.accent3,
```
Comment (veto221) [inline components/Search/styles.js:47]: Nice :) Better for me is using predefined colours instant of hash

File: components/Search/styles.js [hunk 21]
```diff
@@ -64,27 +64,27 @@ const styles = StyleSheet.create({
...
     searchResultContainer: {
         paddingTop: 10,
         justifyContent: 'center',
         marginLeft: 30,
     },
     searchResultText: {
-        fontFamily: 'Roboto-Regular',
-        color: '#989898',
+        fontFamily: 'Inter-Regular',
+        color: Colors.accent6,
```
Comment (veto221) [inline components/Search/styles.js:76]: Nice :) Better for me is using predefined colours instant of hash

File: screens/ContentTypeObjectsScreen/styles.js [hunk 22]
```diff
@@ -9,14 +10,14 @@ const styles = StyleSheet.create({
         backgroundColor: 'transparent',
     },
     title: {
-        fontFamily: 'Roboto-Medium',
+        fontFamily: 'Inter-Medium',
         fontSize: 18,
-        color: '#000000',
+        color: Colors.accent3,
```
Comment (veto221) [inline screens/ContentTypeObjectsScreen/styles.js:15]: Nice :) Better for me is using predefined colours instant of hash

File: screens/ContentTypesScreen/styles.js [hunk 23]
```diff
@@ -18,14 +19,14 @@ const styles = StyleSheet.create({
...
         fontSize: 18,
-        color: '#000000',
+        color: Colors.accent3,
     },
     subtitle: {
-        fontFamily: 'Roboto-Regular',
+        fontFamily: 'Inter-Regular',
         fontSize: 15,
-        color: '#333333',
+        color: Colors.accent4,
```
Comment (veto221) [inline screens/ContentTypesScreen/styles.js:29]: Nice :) Better for me is using predefined colours instant of hash

File: screens/HomeScreen/HomeScreen.js [hunk 24]
```diff
@@ -54,7 +54,7 @@ const HomeScreen = (props) => {
                         title="Remove API Key"
                         onPressBtn={() => deleteApiKey()}
                         buttonStyle={styles.btnDanger}
-                        titleStyle={{ color: '#000000' }}
+                        titleStyle={{ color: Colors.accent3 }}
```
Comment (veto221) [inline screens/HomeScreen/HomeScreen.js:57]: Nice :) Better for me is using predefined colours instant of hash

File: screens/HomeScreen/styles.js [hunk 25]
```diff
@@ -24,15 +24,15 @@ const styles = StyleSheet.create({
     },
     h2Style: {
         fontSize: 24,
-        color: '#000000',
+        color: Colors.accent3,
```
Comment (veto221) [inline screens/HomeScreen/styles.js:27]: Nice :) Better for me is using predefined colours instant of hash

File: screens/HomeScreen/styles.js [hunk 26]
```diff
@@ -24,15 +24,15 @@ const styles = StyleSheet.create({
...
         textAlign: 'left',
-        fontFamily: 'Roboto-Regular',
+        fontFamily: 'Inter-Regular',
     },
     paragraph: {
         fontSize: 20,
-        color: '#000000',
+        color: Colors.accent3,
```
Comment (veto221) [inline screens/HomeScreen/styles.js:33]: Nice :) Better for me is using predefined colours instant of hash

File: screens/ObjectScreen/ObjectScreen.js [hunk 27]
```diff
@@ -151,7 +151,6 @@ const ObjectScreen = (props) => {
     const renderItem = (item) => {
         const def = getDefinitionData(contentTypesDefinitions)[ctoName];
         const order = ['id', ...def.order, 'internal'];
-        console.log(def.definitions);
```
Comment (veto221) [inline screens/ObjectScreen/ObjectScreen.js:154]: This is on current production?

File: screens/SearchResultsObjectsScreen/styles.js [hunk 28]
```diff
@@ -12,14 +13,14 @@ const styles = StyleSheet.create({
         paddingVertical: 80,
     },
     title: {
-        fontFamily: 'Roboto-Medium',
+        fontFamily: 'Inter-Medium',
         fontSize: 18,
-        color: '#000000',
+        color: Colors.accent3,
```
Comment (veto221) [inline screens/SearchResultsObjectsScreen/styles.js:18]: Nice :) Better for me is using predefined colours instant of hash

File: screens/SearchResultsObjectsScreen/styles.js [hunk 29]
```diff
@@ -12,14 +13,14 @@ const styles = StyleSheet.create({
...
     },
     subtitle: {
-        fontFamily: 'Roboto-Regular',
+        fontFamily: 'Inter-Regular',
         fontSize: 16,
-        color: '#333333',
+        color: Colors.accent4,
```
Comment (veto221) [inline screens/SearchResultsObjectsScreen/styles.js:23]: Nice :) Better for me is using predefined colours instant of hash

**Comment by veto221** [comment]
There are: 18 vulnerabilities (9 moderate, 9 high)
Maybe not there but we should fix this

**Comment by veto221** [comment]
IMG_1713
Warning (iOS)

**Comment by veto221** [comment]
App not run with react-native-web, 
Should we use this package? 
This give us possibility to run mobile app in web

**Comment by veto221** [comment]
Screenshot 2022-04-21 at 13 29 19

**Comment by veto221** [comment]
As I see on the "Login scree" - where we should enter api key - Flotiq Read-Only API Key text we should change this to Flotiq API Key. 
When we choose read-only than we cannot edit values but app can do it and it work good :)

**Comment by veto221** [comment]
There are: 27 vulnerabilities (12 moderate, 11 high, 4 critical)

**Comment by CiotkaCierpienia** [comment]
> Screenshot 2022-04-21 at 13 29 19

What were you doing before this warning appeared, what iPhone do you have, and on which screen did it happen, that is not enough info for a bug report?

**Comment by CiotkaCierpienia** [comment]
> There are: 27 vulnerabilities (12 moderate, 11 high, 4 critical)

This is outside of the scope of this MR.

File: components/CustomListItem/CustomListItem.js (hunk 4 above)
Comment (CiotkaCierpienia) [reply components/CustomListItem/CustomListItem.js]: CHanged to predefined colour.

**Comment by veto221** [comment]
> > Screenshot 2022-04-21 at 13 29 19
> 
> What were you doing before this warning appeared, what iPhone do you have, and on which screen did it happen? That is not enough info for a bug report.

iPhone 12 (my own, not by simulator) 
 
Tomorrow I will check this one more time and write steps there

**Comment by veto221** [comment]
imageimageimage

**Comment by veto221** [comment]
> > > Screenshot 2022-04-21 at 13 29 19
> > 
> > 
> > What were you doing before this warning appeared, what iPhone do you have, and on which screen did it happen? That is not enough info for a bug report.
> 
> iPhone 12 (my own, not by simulator)
> 
> Tomorrow I will check this one more time and write steps there

This appear when I go on edit page

**Comment by veto221** [comment]
> > > > > Screenshot 2022-04-21 at 13 29 19
> > > > 
> > > > 
> > > > What were you doing before this warning appeared, what iPhone do you have, and on which screen did it happen? That is not enough info for a bug report.
> > > 
> > > 
> > > iPhone 12 (my own, not by simulator)
> > > Tomorrow I will check this one more time and write steps there
> > 
> > 
> > This appear when I go on edit page
> 
> Can you check if it appears on the master branch, and if yes, can you put the issue, as it would be outside the scope of this? If it was introduced in this PR I will try to resolve this problem before merging.
It's on master too, it's out of scope, I accept

**Pull request merged**